juniper_official/Interfaces/interface-throughput.playbook

```diff
@@ -0,0 +1,14 @@
+/*
+ * Playbook contains multiple rules which monitor interfaces bandwidth 
+ * utilization and notifies when anomalies are found.
+ * 
```

Comment:
s/multiple rules/a rule/g
s/when anomalies are found./anomalies when exceeds threshold./g

Reply:
updated the rule